velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -1005,7 +1052,23 @@ TEST_F(Re2FunctionsTest, likeRegexLimit) {
 
   // Over 20 all optimized, will pass.
   for (int i = 0; i < 26; i++) {
-    std::string localPattern = fmt::format("{}.*{}", 'c' + i, 'c' + i);
+    std::string localPattern;
+    // Generate optimized patterns of type kFixed, kPrefix, kSuffix, and
```

Comment:
if one if happened to be not optimized but compiled it wont make this test fail
because 26/4 <20. 
we need to run the test maybe for each of them? maybe we can have a function and call it for each 
also would be nice to add "%%%" to make sure this one is also always optimize either here or some other place

Reply:
Introduced a lambda to run the test for each case 26 times, thanks for the feedback.